src/site/facilities/service_location.drupal.liquid

```diff
@@ -103,14 +103,16 @@
     </div>
   {% endif %}
 
-  {% if locationEntity.fieldReferralRequired and locationEntity.fieldReferralRequired != 'not_applicable' %}
+  {% if locationEntity.fieldReferralRequired and locationEntity.fieldReferralRequired != 'not_applicable' or
+      if locationEntity.fieldReferralRequired and locationEntity.fieldReferralRequired != 'unknown' %}
     <p class="vads-u-margin-bottom--0" data-template="paragraphs/service_location">
       <strong>Referral required?</strong>
       {% if locationEntity.fieldReferralRequired == '1' %}Yes{% else %}No{% endif %}
     </p>
   {% endif %}
 
-  {% if locationEntity.fieldWalkInsAccepted and locationEntity.fieldWalkInsAccepted != 'not_applicable' %}
+  {% if locationEntity.fieldWalkInsAccepted and locationEntity.fieldWalkInsAccepted != 'not_applicable'or
+      if locationEntity.fieldReferralRequired and locationEntity.fieldReferralRequired != 'unknown' %}
```

Comment:
I think you should be using `fieldWalkInsAccepted` here.

Also: I have a suggestion that I think makes this a bit easier to read:

```liquid
  {% assign SHOW_REFERRAL_OR_WALKIN = "not_applicable, unknown" %}

  {% if locationEntity.fieldReferralRequired %}
    {% unless SHOW_REFERRAL_OR_WALKIN contains locationEntity.fieldReferralRequired %}
      <p class="vads-u-margin-bottom--0" data-template="paragraphs/service_location">
        <strong>Referral required?</strong>
        {% if locationEntity.fieldReferralRequired == '1' %}Yes{% else %}No{% endif %}
      </p>
    {% endunless %}
  {% endif %}

  {% if locationEntity.fieldWalkInsAccepted %}
    {% unless SHOW_REFERRAL_OR_WALKIN contains locationEntity.fieldWalkInsAccepted %}
      <p class="vads-u-margin-bottom--0" data-template="paragraphs/service_location">
        <strong>Walk-ins accepted?</strong>
        {% if locationEntity.fieldWalkInsAccepted == '1' %}Yes{% else %}No{% endif %}
      </p>
    {% endunless %}
  {% endif %}
```
Sadly, liquid doesn't support assigning arrays to variables (!?!), and the `contains` operator only works on strings. So this is a little hacky, but I think it's preferable.